### ISSUE TITLE
the PS_sp and SD2_sp training sets separate or together during training

### ISSUE THREAD

**Issue by zhangyq0711** [IDLabMedia/tgif-dataset#2]
May I ask if you used only the PS_sp training set or included the SD2-sp dataset when comparing the performance of the PS_sp dataset on different tampering localization models?  Looking forward your reply

**Comment by hmareen** [comment]
Hi!

I am a bit confused about your question. In the paper, in the benchmark of Table I, the PS-sp subset only uses the test set of PS_sp. So not the training set, and not SD2_sp.

The SD2-sp subset (separate column in the table) uses the SD2_sp test set.

Hope this clarifies your question.

Kind regards,
Hannes
